scene/gui/tree.cpp

```diff
@@ -4800,6 +4926,46 @@ int Tree::get_item_offset(TreeItem *p_item) const {
 	return -1; //not found
 }
 
+int Tree::_get_item_h_offset(TreeItem *p_item) const {
+	TreeItem *it = root;
+	int nestingLevel = 0;
```

Comment:
We use snake_case only in Godot:

```suggestion
	int nesting_level = 0;
```